concepts/Common/Security Considerations/20 HTML Encoding/00 HTML Encoding.md

```diff
@@ -0,0 +1 @@
+HTML encoding is a simple technique that helps protect your web application from <a href="https://en.wikipedia.org/wiki/Cross-site_scripting" target="_blank">cross-site scripting (XSS)</a> attacks. In an XSS attack, the attacker injects a malicious code script into your web application. Every time a user visits the infected part of the application, this script runs. To prevent the code injection, user input must always be encoded (converted to plain text).
```

Comment:
```suggestion
HTML encoding is a simple technique that helps protect your web application from <a href="https://en.wikipedia.org/wiki/Cross-site_scripting" target="_blank">cross-site scripting (XSS)</a> attacks. In an XSS attack, the attacker injects a malicious script into your web application. Every time a user visits the infected part of the application, this script runs. To prevent the code injection, user input must always be encoded (converted to plain text).
```